include/deal.II/numerics/tensor_product_matrix_creator.h

```diff
@@ -0,0 +1,356 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2022 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_tensor_product_matrix_creator_h
+#define dealii_tensor_product_matrix_creator_h
+
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/fe/fe.h>
+
+#include <deal.II/grid/tria.h>
```

Comment:
I guess it gets transitively included, but I would prefer if we could also `#include <deal.II/base/quadrature.h>`